medusa/cassandra_utils.py

```diff
@@ -340,15 +357,15 @@ def hostname(self):
 
     @property
     def storage_port(self):
-        return self._storage_port
+        return int(self._storage_port)
```

Comment:
yup, good call 👍

Reply:
I believe this is the source of the issues in the integration tests failing in this PR. I will refactor those to assist @ivanmp91 as well with his #320.  I had requested he do the same, and I see some integration test failures there that may be working with string types still for ports.

Reply:
Added an integration test fix for this that takes care of the other PR issues as well.  I'll push that to #320 for @ivanmp91.